cmd/osbuild-worker/jobimpl-koji-finalize.go

```diff
@@ -305,13 +311,16 @@ func (impl *KojiFinalizeJobImpl) Run(job worker.Job) error {
 		}
 	}
 
+	if args.StartTime > math.MaxInt64 {
```

Comment:
when would that happen? StartTime _is_ uint64?

Reply:
Right, but it's comparing with max *int* 64 (not u).

Reply:
You are totally right - I overlooked that :-/